### PR TITLE
Updated the danish language spelling errors

### PR DESCRIPTION
**Pull request by larskochhansen**
I've correct danish spelling errors and added a few missing translations.
Best.

### CODE REVIEW

File: priv/gettext/da/LC_MESSAGES/default.po [hunk 1]
```diff
@@ -572,7 +572,7 @@ msgstr "Bagklap åben"
 #: lib/teslamate_web/live/charge_live/cost.html.leex:112
 #: lib/teslamate_web/live/geofence_live/form.html.leex:66
 msgid "Per Minute"
-msgstr ""
+msgstr "Pr. Minute"
```
Comment (larserhardsen) [inline priv/gettext/da/LC_MESSAGES/default.po:575]: Pr minut?

File: priv/gettext/da/LC_MESSAGES/default.po [hunk 2]
```diff
@@ -582,12 +582,12 @@ msgstr "Softwareopdatering tilgængelig (%{version})"
...
 #: lib/teslamate_web/live/signin_live/index.html.leex:70
 msgid "Enter the verification code generated by your authenticator app."
-msgstr ""
+msgstr "Indtast verifikationskoden genereret af din authentikator app."
 
 #, elixir-format
 #: lib/teslamate_web/live/signin_live/index.html.leex:50
 msgid "Two-Factor Authentication"
-msgstr ""
+msgstr "To-faktor authentikation"
```
Comment (larserhardsen) [inline priv/gettext/da/LC_MESSAGES/default.po:590]: To-faktor autentifikation

File: priv/gettext/da/LC_MESSAGES/default.po (hunk 1 above)
Comment (larskochhansen) [inline priv/gettext/da/LC_MESSAGES/default.po:575]: Pr. minut

File: priv/gettext/da/LC_MESSAGES/default.po (hunk 2 above)
Comment (larskochhansen) [reply priv/gettext/da/LC_MESSAGES/default.po:590]: Agree. I'm not completely sure how to edit the pull request other than making a new one.

**Review by larskochhansen — COMMENTED** [review]
The to suggested changes are correct.

**Review by adriankumpf — APPROVED** [review]
I just added the two suggested changes. The PR looks good to me. 

Thank you @larskochhansen!

**Pull request merged**